src/mongo/delegates/Views.class.php

```diff
@@ -19,7 +19,7 @@ class Views extends CompositeBase
      * Construct accepts actual objects rather than strings as this class is a delegate of
      * Tripod and should inherit connections set up there
      * @param string $storeName
-     * @param \MongoCollection $collection
+     * @param \MongoCollection $collection // TODO: should this really be a constructor arg?
```

Comment:
@junglebarry, what's the use case for instantiating a View without going through `Tripod`?  How would you deal with a `view_specifications` config where the various specs had different `to` properties?  We don't do this, currently, in views, but we do in tables.  I'm not sure I understand how you'd use a `View` object that wasn't rooted in a CBD originally.